### PR TITLE
Create flexbox-aligning-flex-items.md

### PR DESCRIPTION
**Pull request by leebut**
This file describes and illustrates how flex-direction, justify-content, align-items and align-content work together.
I will take some screenshots for the align-content section. All other screenshots are done.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by leebut** [comment]
I don't understand why this failed.

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Please either replace all the alt text with correct ones or leave them empty

**Comment by leebut** [comment]
I don't understand. What alt text? I didn't include any images.

File: guide/english/css/flexbox-how-to-align-flex-items/index.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+There is a short code sample at the end of this page.
+
+![alt text](https://raw.githubusercontent.com/leebut/Flexbox-Images/master/main%20axis%20row.png "Flex direction, row.")
+
+![alt text](https://raw.githubusercontent.com/leebut/Flexbox-Images/master/main%20axis%20col.png "Flex diection, column.")
+
+![alt text](https://raw.githubusercontent.com/leebut/Flexbox-Images/master/cross%20axis%20row.png "Cross axis of a row.")
+
+![alt text](https://raw.githubusercontent.com/leebut/Flexbox-Images/master/cross%20axis%20col.png "Cross axis of a column.")
```
Comment (thecodingaviator) [inline guide/english/css/flexbox-how-to-align-flex-items/index.md]: These^

**Comment by RandellDawson** [comment]
@thecodingaviator This one has gone stale.

**Comment by thecodingaviator** [comment]
Hi @leebut 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**